x/gamm/pool-models/balancer/pool_test.go

```diff
@@ -33,131 +33,97 @@ var (
 // TestUpdateIntermediaryPoolAssetsLiquidity tests if `updateIntermediaryPoolAssetsLiquidity` returns poolAssetsByDenom map
 // with the updated liquidity given by the parameter
 func TestUpdateIntermediaryPoolAssetsLiquidity(t *testing.T) {
-	testCases := []struct {
-		name string
-
-		// returns newLiquidity, originalPoolAssetsByDenom, expectedPoolAssetsByDenom
-		setup func() (sdk.Coins, map[string]balancer.PoolAsset, map[string]balancer.PoolAsset)
+	const (
+		uosmoValueOriginal = 1_000_000_000_000
+		atomValueOriginal  = 123
+		ionValueOriginal   = 657
```

Comment:
Do you think it makes sense to have one test case with initial pool assets other than these?